Automattic/wp-calypso
# calypso-products: Remove Backup/Security (1TB) variants

This PR removes all Jetpack VaultPress Backup and Jetpack Security product variants from the map of Jetpack related products, which effectively removes the choice between 10GB and 1TB plans from the Jetpack.com pricing page (defaulting to 10GB only).

_(NOTE: All existing subscriptions should still behave as they do in production, meaning all "Manage" buttons are still present and redirect to the proper pages.)_

Resolves `1161179020265237-as-1203944276482158`.

## Proposed Changes

* In the `@automattic/calypso-products` package, remove the following product slugs and arrays of related products from `JETPACK_RELATED_PRODUCTS_MAP`:
  * `PRODUCT_JETPACK_BACKUP_T1_YEARLY`
  * `PRODUCT_JETPACK_BACKUP_T2_YEARLY`
  * `PLAN_JETPACK_SECURITY_T1_YEARLY`
  * `PLAN_JETPACK_SECURITY_T2_YEARLY`

## Testing Instructions

Ensuring you're within the context of this pull request:

1. Without being logged in to Jetpack.com or WordPress.com, visit the Jetpack.com pricing page. Verify Security and VaultPress Backup are present on the page, that clicking on their "More about `X`" links brings up a detailed lightbox, and that only the 10GB variant of these products is displayed (i.e., no radio buttons).
2. Log into Jetpack.com and repeat the instructions from step 1. (The behavior should be identical).
3. Add a site you own into your viewing context by adding its slug at the end of the browser URL (e.g., `/pricing/mysite.example`). Verify that if you have either VaultPress Backup or Security (either 10GB or 1TB), you see the correct "Manage plan," "Manage subscription," or "Upgrade to Yearly" buttons in the appropriate product card(s).

## Reference screenshot

<img width="1119" alt="image" src="https://user-images.githubusercontent.com/670067/223774343-60eae985-4430-47b6-aeff-a21b26657196.png">

<img width="1116" alt="image" src="https://user-images.githubusercontent.com/670067/223774384-b37ab313-544f-46c9-898e-657169566eda.png">

## Pre-merge Checklist




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203944276482158
